docs/getting-started.md

```diff
@@ -0,0 +1,44 @@
+# Getting started 
+
+## What is Luigi?
+
+Luigi is an open source JavaScript framework for micro frontends. It enables you to:
+* create a unified user interface or web application using **Luigi Core**. 
+* create micro frontends and connect them to the Luigi Core app using **Luigi Client**.
+
+## What are micro frontends? 
+
+Micro frontends are small frontend web applications that are implemented on one global user interface. They allow you to:
+
+* **break up big, monolithic UIs**: decompose your frontend into smaller chucks and make it more manageable, all while maintaining a consistent user experience.
+* **give individual teams autonomy**: every team in your project can develop their own end-to-end application which exists on the same core app. This gives teams full autonomy over release dates and updates, while also reducing code dependencies.
```

Comment:
You're right, because otherwise we repeat "application" twice in the same sentence. 😃 

I'm not sure about `This gives teams full autonomy over release dates and updates, at the same time reducing code dependencies.` There should be a word like "while" somewhere (a subordinating conjunction) because "at the same time reducing code dependencies" isn't a standalone sentence. 
Maybe just `This gives teams full autonomy over release dates and updates, and at the same time reduces code dependencies.`
Or `This gives teams full autonomy over release dates and updates, while at the same time reducing code dependencies.`?

Reply:
You are right  - I like the second one :) 👍